util/mod_translation_updater.py

```diff
@@ -121,9 +120,9 @@ def main():
 
 # Group 2 will be the string, groups 1 and 3 will be the delimiters (" or ')
 # See https://stackoverflow.com/questions/46967465/regex-match-text-in-either-single-or-double-quote
-pattern_lua_quoted = re.compile(r'[\.=^\t,{\(\s]N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
+pattern_lua_quoted = re.compile(r'(?:^|[\.=^\t,{\(\s])N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
```

Comment:
question: why do we even need the `(?:^|[\.=^\t,{\(\s])` ?

Reply:
answering my own question: this is needed so it matches `S(` but not `BOATS(`
`\b` would be the more obvious way to do this (if it works)